web/src/local-storage.ts

```diff
@@ -0,0 +1,48 @@
+export const RTDM_LOCAL_STORAGE_KEY = "rtdm-app";
+
+type LocalStorage = {
+  userID: string | undefined;
+};
+
```

Comment:
Can you use recoil to access local storage rather than introducing a separate access pattern?